WeaponPack/src/main/java/com/github/sef24sp4/weaponpack/shotgun/ShotGun.java

```diff
@@ -29,11 +29,43 @@ public boolean shoot(IEntityManager entityManager, IEntity shooter) {
 		return true;
 	}
 
+	//ProjectileSpreader method
+	public IEntity projectileSpreader(IEntityManager entityManager, IEntity shooter) {
+		//StaggeredValue fortæller, hvordan vores projektiler er forskudte fra hinanden.
+		//Dette er en eksperiment.
+		ShotGunBullet shotGunBullet = new ShotGunBullet(shooter);
+		double staggeredValue = Math.PI / 4;
+		//Random er brugt til at fortælle, at der skal skyde/spawne 25 projektiler når vi skyder.
+		Random random = new Random();
+		double spreadValue = random.nextInt(0, 5);
+		for (int i = 0; i < spreadValue; i++) {
+			//Mening med staggeredPosition er at skabe afvigelsen af positionen af projektilerne.
+			double staggeredPosition = this.staggeredProjectiles(staggeredValue);
+			//Efter skaber vi changedRotation, hvor vi forsøger at definere den nye changedRotation.
+			double changedRotation = shooter.getRotation() + random.nextDouble(staggeredPosition / 2, staggeredPosition);
+			//Denne her er meget sjov, fordi her sætter jeg shooterens rotation selvom det burde være projektilet.
+			shotGunBullet.setRotation(changedRotation);
+			//Projektilet er tilføjet og det gør at vi skyder.
+			entityManager.addEntity(this.shotGunBulletControlSystem.createProjectile(shooter));
+		}
+		return shooter;
+	}
+
+	//staggeredProjectiles method
+	public double staggeredProjectiles(double staggeredValue) {
+		Random randomly = new Random();
+		//Hensigten med return-statement er at kunne skabe forskydningen således at vi kan danne et interval.
+		//Arbejdet er stadigvæk igang.
+		return (randomly.nextDouble() + 1) * staggeredValue;
+	}
+
 	private List<ShotGunBullet> createBullets(final IEntity shooter){
 		List<ShotGunBullet> projectiles = new ArrayList<>();
 		//Genererering flere projektiler på samme tid.
 		for (int i = 0; i < ammountOfBulletsPerShot; i++) {
-			this.shotGunBulletControlSystem.createProjectile(shooter);
+			double changedRotation = getRandomRotation(shooter);
+			ShotGunBullet projectile = new ShotGunBullet(shooter);
+			projectiles.add(projectile);
 		}
 		return projectiles;
```

Comment:
Requires Feedback, is it going on the right direction? Pretty sure, that generating projectiles is. Is it suggested, that I create an object or should I remove it and replace it with this.bulletControlSystem.createProjectile(shooter);